pkg/k8s/endpointsynchronizer/cep.go

```diff
@@ -254,17 +256,37 @@ func (epSync *EndpointSynchronizer) RunK8sCiliumEndpointSync(e *endpoint.Endpoin
 					}
 				}
 
-				// We have an object to reuse. Update and push it up. In the case of an
-				// update error, we retry in the next iteration of the controller using
-				// the copy returned by Update.
-				scopedLog.Debug("Updating CEP from local copy")
-				// Update the copy of the cep
-				mdl.DeepCopyInto(&localCEP.Status)
 				switch {
-				case ciliumUpdateStatusVerConstr.Check(k8sServerVer):
```

Comment:
Why are we removing the UpdateStatus case? It would help on k8s 1.11 and 1.12.

Reply:
@tgraf It's still there on line 285